engine.py

```diff
@@ -713,7 +717,7 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
         #       it to work and instead resorted to the event watcher setup.
 
         # make a unique id for the app widget based off of the panel id
-        widget_id = "panel_%s" % panel_id
+        widget_id = tk_maya.panel_generation.SHOTGUN_APP_PANEL_PREFIX + panel_id
```

Comment:
for even better encapsulation, would it make sense to have a `tk_maya.panel_generation.generate_widget_id(panel_id)` - or is that overkill?